x-pack/plugins/monitoring/public/components/elasticsearch/node/advanced.js

```diff
@@ -54,7 +54,7 @@ export const AdvancedNode = ({ nodeSummary, metrics, alerts, ...props }) => {
           <NodeDetailStatus stats={nodeSummary} alerts={alerts} />
         </EuiPanel>
         <EuiSpacer size="m" />
-        <AlertsCallout alerts={alerts} />
+        <AlertsCallout alerts={alerts} stateFilter={(state) => state.nodeId === nodeId} />
```

Comment:
We don't need `stateFilter` for `<AlertCallout />` anymore, since it's only used by node detail pages, which already get the filtered alerts from the server